lib/renderer/security-warnings.js

```diff
@@ -77,204 +77,200 @@ const isUnsafeEvalEnabled = function () {
+
+  console.warn('%cElectron Security Warning (enableBlinkFeatures)',
+    'font-weight: bold;', warning)
+}
+
+/**
+ * #11 on the checklist: Do Not Use allowpopups
+ *
+ * Logs a warning message about allowed popups
+ */
+const warnAboutAllowedPopups = function () {
+  if (document && document.querySelectorAll) {
+    const domElements = document.querySelectorAll('[allowpopups]')
+
+    if (!domElements || domElements.length === 0) {
       return
     }
 
-    const warning = `This renderer process has additional "enableBlinkFeatures"
-    enabled. This exposes users of this app to some security risk. If you do not
-    need this feature, you should disable it.\n ${moreInformation}`
+    const warning = `A <webview> has "allowpopups" set to true. This exposes
+    users of this app to some security risk, since popups are just
+    BrowserWindows. If you do not need this feature, you should disable it.\n
+    ${moreInformation}`
 
-    console.warn('%cElectron Security Warning (enableBlinkFeatures)',
+    console.warn('%cElectron Security Warning (allowpopups)',
       'font-weight: bold;', warning)
-  },
-
-  /**
-   * #11 on the checklist: Do Not Use allowpopups
-   *
-   * Logs a warning message about allowed popups
-   */
-  warnAboutAllowedPopups: () => {
-    if (document && document.querySelectorAll) {
-      const domElements = document.querySelectorAll('[allowpopups]')
-
-      if (!domElements || domElements.length === 0) {
-        return
-      }
+  }
+}
 
-      const warning = `A <webview> has "allowpopups" set to true. This exposes
-      users of this app to some security risk, since popups are just
-      BrowserWindows. If you do not need this feature, you should disable it.\n
-      ${moreInformation}`
+// Currently missing since we can't easily programmatically check for it:
+//   #12WebViews: Verify the options and params of all `<webview>` tags
+
+const logSecurityWarnings = function (webPreferences, nodeIntegration) {
+  warnAboutNodeWithRemoteContent(nodeIntegration)
+  warnAboutDisabledWebSecurity(webPreferences)
+  warnAboutInsecureResources()
+  warnAboutInsecureContentAllowed(webPreferences)
+  warnAboutExperimentalFeatures(webPreferences)
+  warnAboutEnableBlinkFeatures(webPreferences)
+  warnAboutInsecureCSP()
+  warnAboutAllowedPopups()
+}
 
-      console.warn('%cElectron Security Warning (allowpopups)',
-        'font-weight: bold;', warning)
+module.exports = function (nodeIntegration) {
+  window.addEventListener('load', function loadHandler () {
+    if (shouldLogSecurityWarnings()) {
+      const { ipcRenderer } = require('electron')
+      const webPreferences = ipcRenderer.sendSync('ELECTRON_BROWSER_GET_LAST_WEB_PREFERENCES')
+      logSecurityWarnings(webPreferences, nodeIntegration)
     }
-  }
 
-  // Currently missing since we can't easily programmatically check for it:
-  //   #12WebViews: Verify the options and params of all `<webview>` tags
+    window.removeEventListener('load', loadHandler)
```

Comment:
You could simply add a third argument to the `.addEventListener()` call:
```js
window.addEventListener('load', function () {...}, {once: true})
```

Reply:
done